### DISCUSSION TITLE
A77F3B4B-0338-418C-A1AB-15939F0C0787

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#33] [General]
# A77F3B4B-0338-418C-A1AB-15939F0C0787

HD 191939, also known as TOI-1339, is a Sun-like star about 175.12 light-years away in the constellation Draco. Between 2020 and 2021, a total of five planets (two jovians and three neptunians) were discovered by the TESS observatory, with an additional superterran's existence currently awaiting confirmation

https://celestia.mobi/addon?item=A77F3B4B-0338-418C-A1AB-15939F0C0787

**Comment by exocritic5457** [comment]
There's already a 6th planet between e and f. Does it have no textures?

**Reply by exocritic5457** [reply]
I would definitely have submitted an update for this by now with those 6+1 planets given I know I can yet it gives me the feeling of stealing someone else's work doing it and I just don't like stealing. It's not like I'm in any rush for this though.

**Reply by DaveBowman2001** [reply]
The author's update (first published on May 16th) is now successfully added here on mobi as of this writing

**Comment by gtfggjbdrff** [comment]
Hello author of this plugin, I suggest that the texture of the sixth gaseous giant be replaced with a new one in the next update. Thank you
